### ISSUE TITLE
Show the student card

### ISSUE THREAD

**Issue by camillebrulotte** [ApplETS/Notre-Dame#162]
**Describe the solution you'd like**
I would probably put it in the "More" page and add a new section with "Student card".

**Comment by MysticFragilist** [comment]
Ahh good idea! Just laying some question to understand some issues:
- How can we get the student card inside the app? Do we add a "scan for a card" kind of camera? or is there an available digital version of a card? 
- Also, for security purpose, if we go for scan a card, I believe an external API is not preferable for security reason as we would store every student card for ETS student, I'm not sure the school would approve right now. So how do we store it? In the cache? that means that on every logout, we lose the picture... 
- And of course, a picture won't be really useful as the main purpose of the card is to open classrooms. So maybe we will have to talk to the IT department for this one.

I believe this has a lot of challenge, but would be really nice to have this feature, I remember Omnivox in cegep did had that feature and it was really nice not having to take your student card everywhere as you would always have it on your app.

**Comment by apomalyn** [comment]
We discussed this with @camillebrulotte this morning. Currently the school doesn't offer any API or way to pull the student card information. But this is in discussion. 

What we would like:
- offer a way to unlock the doors with the application.
- find a way to display the "official" card something that could be used in exams or security purpose

**Comment by MysticFragilist** [comment]
Okay great! This answer most of my questions! So, for now we really depends on the school to either create an API access point for fetching student info (for your second point) and/or to make classrooms unlockable by NFC or some other mechanism that a phone could use (for your first point). We can't really start working on this issue until this is implemented, correct?

**Comment by apomalyn** [comment]
@MysticFragilist exact, we could start to think about the UI/UX or think which technology is needed, but for now that is all
